### ISSUE TITLE
Autocomplete does not function after 0.22

### ISSUE THREAD

**Issue by Smudded** [mattberkowitz/autoclose-html#139]
I checked out commits until I found a working version and it looks like something in this commit breaks autocomplete for me: https://github.com/mattberkowitz/autoclose-html/commit/73cdb1292978c3b491ef5adc24bb108001aebce3

Obviously not everyone else is having this issue, so I'll continue to troubleshoot and see if I can pinpoint the problem and create an issue for anyone else to post findings if they're affected as well.

**Comment by mattberkowitz** [comment]
Do you use a non-US or non-QUERTY keyboard? If so, see #128

**Comment by Smudded** [comment]
Nope, standard keyboard. I did read through that thread though and that issue seems pretty maddening.

**Comment by robmim** [comment]
I had the same problem and i simply checked the Legacy/International Mode and it worked

**Comment by Deikilia** [comment]
I have the same problem, but cant find the Legacy/International Mode

haha, ok, I finally can find it. TY.
